ci-scripts/unit-tests.sh

```diff
@@ -2,108 +2,33 @@
-
-echo "Running schematics unit tests and code coverage for s4om library"
-
-npm --prefix integration-libs/s4om run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for storefinder library"
+EXCLUDE_JEST=storefrontapp,storefrontstyles,schematics,setup
+EXCLUDE_INTEGRATION_LIBS=cdc,cds,digital-payments,epd-visualization,s4om
 
-npx nx test storefinder --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for storefinder library"
-
-npm --prefix feature-libs/storefinder run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for qualtrics library"
-
-npx nx test qualtrics --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for qualtrics library"
-
-npm --prefix feature-libs/qualtrics run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for asm library"
-
-npx nx test asm --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for asm library"
-
-npm --prefix feature-libs/asm run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for tracking"
-
-npx nx test tracking --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for tracking library"
-
-npm --prefix feature-libs/tracking run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for pickup-in-store library"
-
-npx nx test pickup-in-store --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for pickup-in-store library"
-
-npm --prefix feature-libs/pickup-in-store run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for customer-ticketing"
-
-npx nx test customer-ticketing --source-map --no-watch --code-coverage --browsers ChromeHeadless
-
-echo "Running schematics unit tests and code coverage for customer-ticketing"
-
-npm --prefix feature-libs/customer-ticketing run test:schematics -- --coverage
-
-echo "Running unit tests and code coverage for schematics library"
-
-npm --prefix projects/schematics run test --runInBand -- --coverage
-
-echo "Running unit tests and code coverage for eslint-rules"
+echo "-----"
 
-npx nx test eslint-rules --source-map --no-watch --code-coverage --browsers ChromeHeadless
+function run_affected_unit_tests {
+    echo "Running JASMINE unit tests and code coverage for AFFECTED libraries"
+    npx nx affected --target=test --exclude="$EXCLUDE_JEST,$EXCLUDE_INTEGRATION_LIBS" -- --no-watch --source-map --code-coverage --browsers ChromeHeadless
+
+    echo "Running JEST (mostly schematics) unit tests and code coverage for AFFECTED libraries"
+    npx nx affected --target=test-schematics --exclude="$EXCLUDE_INTEGRATION_LIBS" -- --coverage --runInBand
```

Comment:
Overall, the target could perhaps have been called "test-jest".  I don't mean to be picky though, I wonder if this could improve the readability.  Thoughts? Your call in the end.

Reply:
that's fine with me